x-pack/test/security_solution_cypress/es_archives/threat_indicator/data.json

```diff
@@ -1,12 +1,75 @@
 {
   "type": "doc",
   "value": {
-    "id": "_uZE6nwBOpWiDweSth_D",
-    "index": "threat-indicator-0001",
+    "id": "84cf452c1e0375c3d4412cb550bd1783358468a3b3b777da4829d72c7d6fb74f",
+    "index": "filebeat-7.12.0-2021.03.10-000001",
```

Comment:
The mappings for this index is currently 25k LOC; I think you can copy/paste/slightly modify the [FTR mappings](https://github.com/elastic/kibana/blob/master/x-pack/test/functional/es_archives/filebeat/threat_intel/mappings.json) to reduce that file by two orders of magnitude 😉